declarations/event.d.ts

```diff
@@ -3381,13 +3544,6 @@ declare namespace WoWAPI {
          */
         NEUTRAL_FACTION_SELECT_RESULT: Unknown[];
 
-        /**
-         * Fired when a user drags an item into the Auction Item box on the Auctions tab of the auction house window
-         * @description You can retrieve information about the item for which an auction is being considered using GetAuctionSellItemInfo
-         * @see https://wow.gamepedia.com/NEW_AUCTION_UPDATE
-         */
-        NEW_AUCTION_UPDATE: null;
```

Comment:
You are right, this event is not present in 8.3. i will release a classic version to support classic addons.